TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -932,14 +972,11 @@ public static (SymbolType, Node) ResolveTargetSectionOrParagraph(Node callerNode
 
             IList<Section> GetSections() => callerNode.SymbolTable.GetSection(target.Name);
 
-            IList<Paragraph> GetParagraphs() => callerNode.SymbolTable.GetParagraph(target.Name);
+            IList<Paragraph> GetParagraphs() => callerNode.SymbolTable.GetParagraph(target);
         }
 
-        private static void Check<T>(string nodeTypeName, T node, [NotNull] IList<T> found) where T : Node
+        private static void CheckIsEmpty<T>(string nodeTypeName, T node) where T : Node
```

Comment:
Well, the purpose is to check that it is _not_ empty I think.

Reply:
Done